### PR TITLE
Update rajendra.txt

### PR DESCRIPTION
**Pull request by rajendra19950**
i added merged 2 lines in to 1 line.

### CODE REVIEW

**Comment by sso-vineel** [comment]
Make starting letter of the line as Capital.

File: DevFiles/rajendra.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-this is rajendra.
-welcome to ssi.
+this is rajendra.welcome to ssi.
```
Comment (sso-vineel) [inline DevFiles/rajendra.txt]: Make the first letter of the line as Capital

Comment (rajendra19950) [reply DevFiles/rajendra.txt]: i changed to cap letter.

**Review by sso-vineel — APPROVED** [review]
Good Job

**Pull request merged**